src/main/java/com/google/codeu/servlets/UserListServlet.java

```diff
@@ -23,6 +23,9 @@ public void init() {
     datastore = new Datastore();
   }
 
+ /**
+   * Responds with a JSON representation of all users. 
+   */
```

Comment:
Here it would be helpful to include something in the comment like. `Example JSON: ["user1@test.com", "user2@test.com", "user3@example.com"]`.

Reply:
Perfect!